src/server/components/dashboard/helpers.ts

```diff
@@ -57,3 +61,27 @@ export async function getInitiateFormRunnerSessionToken(
   logger.info(`token: ${token}`);
   return token;
 }
+
+export const pageTitles: { [key: string]: string } = {
+  [dashboardRoutes.usersEdit]: "list management - edit user",
+  [dashboardRoutes.usersList]: "list management - user list",
+  [dashboardRoutes.lists]: "list management - all provider lists",
+  [dashboardRoutes.listsEdit]: "list management - :serviceType in :country - edit provider list",
+  [dashboardRoutes.listsItems]: "list management - :serviceType in :country - provider list",
+  [dashboardRoutes.listsItem]: "list management - :serviceType in :country - provider details",
+  [dashboardRoutes.listsItemDelete]: "list management - :serviceType in :country - confirm delete provider",
+  [dashboardRoutes.listsItemPublish]: "list management - :serviceType in :country - confirm publish list item",
+  [dashboardRoutes.listsItemRequestChanges]: "list management - :serviceType in :country - confirm request changes to provider",
+  [dashboardRoutes.listsItemUpdate]: "list management - :serviceType in :country - confirm update provider",
+  [sitemapRoute]: "site map",
+  [authRoutes.login]: "list management - login",
+  [authRoutes.logout]: "list management - logout",
+}
```

Comment:
I have no major qualms about this variable aside from style so please see this as an optional change.

What are your thoughts on the pageTitles variable being an array of objects with a type that resembles the following:

```ts
interface PageTitle {
  url: string,
  title: string
};

type PageTitles = Array<PageTitle>;
```

The disadvantage of this is that extra logic will need to be included in the getPageTitle() function (or any other function of your choosing) to determine which title value to use based on the url.

However, although the code will gain some complexity it the change will greatly aid readability.

It would be good to get your thoughts on this.

Reply:
Yeah for sure, code performance wise I'm sure there's a slight hit since you're running a function for getting the title instead of using an object but as it's running server side and it's not client-side Javascript it's a hit I think that's worth the readability.

My assumption would be the logic would be in a function like `getPageTitle` so to get a title it would look like the following `getPageTitle(dashboardRoutes.lists)`, but it's not that big of a deal. Nothing to hold the PR back over.

Reply:
objects/maps are simplest/cleanest. They are a simpler concept to understand than array functions. 

I suppose you could abstract to `getPageTitle(..)` which is just an object lookup.. but it just looks like abstraction for abstraction's sake. 

having performant server side code is more important. If we max out our resources (RAM or CPU) - service goes down for everyone. If we max out someone's device (if it was an SPA), it would impact a single user. We also pay for the resource.